item-list.html

```diff
@@ -118,70 +118,161 @@ <h2 class="c-section__heading">
   </div>
 
   
+  
+  <!-- コンテンツ読み込みに成功したときは、display:none を外す処理 -->
+  <script>
+    window.addEventListener('spearlyCmsAfterInit', (evt) => {
+      // 取得成功したものについては、display: none を外す
+      const contentId = evt.cmsContentId;
+      const className = `.c-item-list__item-from-spearly-${contentId}`;
+      const targetElement = document.querySelector(className);
+      targetElement.classList.remove(className);
+    })
+  </script>
 
-
+  <!-- 商品リスト -->
   <ul class="c-product-list">
     <{section name=num loop=$productlist}>
+      <!-- Spearly / カラーミー両方とも最初は隠しておきます -->
+      <style>
+        /* コンテンツが読み込めなかった時に表示する要素に当てるクラス */
+        /* backup-* のようにワイルドカードでも問題ありません */
+        .backup-<{$productlist[num].id}> {
+          display: none;
+        }
+        /* コンテンツ読み込みまで表示崩れを防ぐために非表示にするクラス */
+        /* 上のscriptの処理で削除されるクラスです */
+        .c-item-list__item-from-spearly-<{$productlist[num].id}> {
+          display: none;
+        }
+      </style>
+      <!-- 商品個別 (Spearly から) -->
 
-    <li class="c-product-list__item">
-      
-      <{* 商品画像 *}>
-      <a href="<{$productlist[num].link_url}>" class="c-product-list__image-wrap c-image-wrap c-image-wrap--link">
-        <{* 商品画像1枚目 *}>
-        <div class="c-image-box<{if $productlist[num].otherimg[0].url != ""}> c-image-box--main<{/if}>">
-          <{if $productlist[num].img_url != ""}>
-          <img src="<{$productlist[num].img_url}>" alt="<{$productlist[num].name}>" class="c-image-box__image js-lazyload" />
-          <{else}>
-          <span class="c-image-box__image c-image-box__image--noimage">No Image</span>
+      <li 
+        class="c-product-list__item c-item-list__item-from-spearly-<{$productlist[num].id }>"
+        cms-item
+        cms-content-type="color-me-items"
```

Comment:
```suggestion
        cms-content-type="sample-ab-testing"
```

こちらもコンテンツタイプIDの修正です 🙇